### PR TITLE
Add secrethub/exec command

### PR DESCRIPTION
**Pull request by florisvdg**
Will add some examples in a different PR.

### CODE REVIEW

File: src/commands/exec.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    type: string
+    description: Command to execute with secrets
+  version:
+    type: string
+    default: latest
+    description: Version of the SecretHub CLI
+  flags:
+    type: string
+    default: ''
+    description: Flags to pass to the `secrethub run` command
```
Comment (SimonBarendse) [inline src/commands/exec.yml:13]: I believe `--no-prompt` should be enabled by default.

Comment (SimonBarendse) [inline src/commands/exec.yml:13]: Do we want to pass the flags separately, i.e. create a field for each flag?

Comment (florisvdg) [reply src/commands/exec.yml:13]: By default or always?

Comment (florisvdg) [reply src/commands/exec.yml:13]: These fields are limited, no maps or lists are supported. And it's easier to add new fields later on when they do support these types than the other way around.

Comment (SimonBarendse) [reply src/commands/exec.yml:13]: Always :+1:

Comment (SimonBarendse) [reply src/commands/exec.yml:13]: We could make it configurable in it's own parameter if we find a use case.

**Review by SimonBarendse — DISMISSED** [review]

**Review by SimonBarendse — APPROVED** [review]

**Pull request merged**